### ISSUE TITLE
I can not use attachments

### ISSUE THREAD

**Issue by AcetonGamerPlayz** [MrCrayfish/MrCrayfishGunMod#376]

**Comment by MrCrayfish** [comment]
Please be more descriptive of the issue you are having.

**Comment by juliolinehanvt** [comment]
I am also having this issue. I added your (absolutely wonderful) gun mod to main mod pack after already starting a world. Last night I crafted the heavy rifle in survival and tried to attach a long range scope and a stock in a crafting table. Individually and together, dyed or standard, with the gun loaded and unloaded, nothing wanted to attach. I thought it might be the heavy rifle so I crafted a rifle and they would not attach to that either.

Also, pressing J doesn't do anything, I imagine because there are no attachments on the gun, but I thought that might be the way to attach them. I checked the key bind.

I am playing on 1.19.2.
